Makefile

```diff
@@ -45,6 +46,11 @@ else ifeq ($(shell $(PKG_CONFIG) --exists libsystemd && echo "0" || echo "1"), 0
 	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd) -D USE_JOURNALD=0
 endif
 
+ifeq ($(shell $(PKG_CONFIG) --exists libseccomp && echo "0" || echo "1"), 0)
+	override LIBS += $(shell $(PKG_CONFIG) --libs libseccomp) -ldl
+	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libseccomp)
```

Comment:
I think we should have some similar variable USE_SECCOMP=0 and branch on behavior

Reply:
fixed and pushed a new version